gems/aws-sdk-resources/CHANGELOG.md

```diff
@@ -1,13 +1,85 @@
 Unreleased Changes
```

Comment:
This is probably something to revisit - we want this changelog at the top level, despite the disconnect in the dependency chain.

I believe this is also related to the test failure this PR creates.